bootstrap/090_post_setup.sql

```diff
@@ -1,5 +1,5 @@
 
-create or replace procedure admin.finalize_setup()
+create or replace procedure admin.finalize_setup(log_output boolean default true)
```

Comment:
why would this be optional?

Reply:
Ya got me: this is the one part of this PR I wasn't happy with.

I added logging to `admin.upgrade_check` to later capture any errors in that procedure that happen outside of `admin.finalize_setup()`

However, I also didn't want to break someone being able to `call admin.finalize_setup()` nor miss out on the logging if they do call that procedure. However, we need to have some logging in `admin.upgrade_check` as we are doing more than just calling `admin.finalize_setup()`.

Finally, I also didn't want to have two tables to generate logging into.

The "best" solution I came up with was to have `admin.upgrade_check()` call `admin.finalize_setup(false)` to prevent both procedures from generating a log message. I still think it's ugly.

Reply:
I don't understand the problem you are trying to solve w/ the boolean? Why cant both log?

Reply:
I wanted to avoid the upgrade_check task running every 24 hours and generating two rows of output into the same table. If you think this isn't worth the code to maintain it, i'll just drop it.

Reply:
yeah, I don't care at all...